neo4j/src/test/java/org/hibernate/ogm/datastore/neo4j/test/query/nativequery/Neo4jSessionNativeQueryTest.java

```diff
@@ -204,6 +206,35 @@ public void testNativeQueryWithMaxRows() throws Exception {
 	}
 
 	@Test
+	public void testNativeQueryExecuteUpdate() throws Exception {
+		OgmSession session = (OgmSession) openSession();
+		Transaction transaction = session.beginTransaction();
+
+		String findQueryString = "MATCH (n:UPDATE) RETURN n";
```

Comment:
Could you use `UPDATE_TABLE_NAME` instead of `UPDATE`? It took me some time to understand it was the table name :).